proparse/src/main/java/org/prorefactor/proparse/ParserSupport.java

```diff
@@ -287,28 +285,29 @@ SymbolScope parseSuper(JPNode classNode, String qualSuperName)
       throws IOException, ANTLRException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)